### PR TITLE
CI: parse examples

### PR DESCRIPTION
**Pull request by stsewd**
This is currently failing, we need to fix the grammar to make it work with those files, but this give us a nice way to see what's missing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+	npm run build
+
+test: build
+	npm run test
 
 run: all
-	tree-sitter build-wasm
-	tree-sitter web-ui
+	npm run wasm
+	npm run web
```
Comment (stsewd) [inline Makefile:11]: replaced all these, so they are run using the tree-sitter binary from the project, and not the global one.

**Comment by stsewd** [comment]
I think this can be merged and have the job as not required till we fix all the cases

**Comment by vigoux** [comment]
I think we shoudl have a pile of files in an `examples` directory, of even in `corpus` that are just used for this IMO.

**Comment by vigoux** [comment]
Thanks to the @Fymyte 's work in #47 and #49 , we can now parse the `syntax/vim.vim`, and a bunch of other files as well. This is thus making progress slowly.

**Pull request closed without merging**